frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -155,6 +157,22 @@ export const EmbedFrame = ({
   const shouldApplyParameterPanelThemeChangeTransition =
     !isParameterPanelStickyStateChanging && isParameterPanelSticky;
 
+  const valuePopulatedParameters = parameters
```

Comment:
Since this value is moved outside the JSX which was under `hasVisibleParametersCondition`. It wouldn't crash before as `parameters` would be present at the time.